packages/app/components/feed/feed.md.tsx

```diff
@@ -162,109 +155,69 @@ export const FeedList = () => {
 // };
 
 const HomeFeed = () => {
-  const { data } = useFeed();
-
-  return <NFTScrollList data={data} fetchMore={() => null} />;
+  const { data, isLoading } = useFeed();
+  return (
+    <NFTScrollList data={data} fetchMore={() => null} isLoading={isLoading} />
+  );
 };
 
-const NFTScrollList = ({
-  data,
-  fetchMore,
-}: {
+type NFTScrollListProps = {
   data: NFT[];
-  fetchMore: any;
+  fetchMore: () => void;
+  isLoading: boolean;
   // tab?: Tab;
-}) => {
-  const { width: screenWidth, height } = useWindowDimensions();
-  let dataProvider = useMemo(
-    () =>
-      new DataProvider((r1, r2) => {
-        return r1.nft_id !== r2.nft_id;
-      }).cloneWithRows(data),
-    [data]
-  );
-
-  const _layoutProvider = useMemo(
```

Comment:
mah man. This API used to give me PTSD 😌

Reply:
+1!! especially on the web, too many weird issues!
I'm very empathetic and battle with this API! 🤣